test/unit/entrypoint/test_main.py

```diff
@@ -13,9 +14,21 @@ def entrypoint_method(name: str) -> str:
 
 def test_no_args(mocker):
     mocker.patch("sys.argv", ["app"])
-    mocker.patch(entrypoint_method("sleep_inifinity"))
+    mocker.patch(entrypoint_method("sleep_infinity"))
     entrypoint.main()
-    assert entrypoint.sleep_inifinity.called
+    assert entrypoint.sleep_infinity.called
+
+
+def test_user_arg(mocker):
+    mocker.patch("sys.argv", ["app", "--user", "jennifer", "--group", "heroes"])
+    user = create_autospec(entrypoint.User)
+    user.own.return_value = user
+    mocker.patch(entrypoint_method("User"), return_value=user)
+    mocker.patch(entrypoint_method("sleep_infinity"))
+    entrypoint.main()
+    assert user.own.called
```

Comment:
I think, this test is missing an assert for the constructor of User

Reply:
See next push